public/js/src/client.js

```diff
@@ -107,6 +119,11 @@ $(document).ready(function(){
 			},
 			up: function(){
 				var c = allChunks[this.location];
+// I wouldn't use jQuery for this.  I'd cache the elem the first time I looked it up:
+// c.element = c.element || document.getElementById(c.id);
+// Then use c.element.offsetLeft and c.element.offsetTop
+// If you tried to cache the jQuery object, on the other 
+// hand, you'd cause a big ol' drain on the RAM.
```

Comment:
How big of a drain on ram are we talking?  There are going to be a few hundred of these objects... The one thing I keep thinking about with this is that the page is pretty small relative to the amount of it that needs to be accessible by Jquery.  We should talk about this.